tests/scripts/wait-for-alfresco-start.sh

```diff
@@ -24,6 +24,7 @@ if (("$COUNTER" < "$TIMEOUT")) ; then
    t1=$(date +%s)
    delta=$((($t1 - $t0)/60))
    echo "Alfresco Started in $delta minutes"
+   sleep 180
```

Comment:
Please add a comment here or modify the message to say that Alfresco started successfully in x minutes, waiting for all the containers to initialise", or something along that. And a message after the 3 minutes end.

Also add a short description to the PR.